### PR TITLE
Article 046: Overflow

### PR DESCRIPTION
**Pull request by Fjolyo**
#### Length of text ####
:page_with_curl: short

####  Difficulty to read ####
:mouse:  light

####  Difficulty to understand #### 
:school_satchel:  easy

####  Type of text #### 

:bowtie:  informative

####  Type of header #### 
 
:bulb:  informative

### CODE REVIEW

**Comment by Teifun2** [comment]
**Plenum:**
Problem:
- Mechanical Scale as the example?
- Problem with the first image, not very much connected.
- The Spacerocket had no overflow rather a casting error.

**Comment by MoiraZuber** [comment]
Added the new image. Kept it close to the odometer image, but did not find the "+1" and "overflow" necessary. Can be added if considered important.

File: articles/article-046-overflow.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!-- BEGIN TITLE -->
+# Overflow
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Imagine an odometer of a car, there are six spaces each of which can display a one digit number. Therefore, the highest number that can be represented is 999'999, after which the odometer turns to 000'000 again.
```
Comment (marcelluethi) [inline articles/article-046-overflow.md]: Language suggestion:  Imagine an odometer of a car. There are ...

**Comment by Fjolyo** [comment]
> * Mechanical Scale as the example?

I think the odometer, especially with the picture, should be clear enough.

> * Problem with the first image, not very much connected.

Changed to use the second picture.

> * The Spacerocket had no overflow rather a casting error.

I think it's still okay to call it an overflow error. Everyone else does it as well and it shows the severity of such errors.

**Comment by Fjolyo** [comment]
> I like the article. I only have this minor remark regarding the language in the first sentence.

Fixed that.

**Comment by noverney** [comment]
- can add a hyphen between "one digit"
- can change "As soon as all of these bits are set" to "As soon as all these bits are set"

**Comment by SimonDold** [comment]
The text says "There are SIX spaces each of which can display a one digit number." but the picture shows 8. 
This should be consistent.

**Comment by marcelluethi** [comment]
@Fjolyo Please change the text to SIX spaces (see above comment from @SimonDold), as this is easier than changing the image

**Comment by marcelluethi** [comment]
great, once you incorporated the two small comments of Normand, the article will be finished.

**Comment by Fjolyo** [comment]
> great, once you incorporated the two small comments of Normand, the article will be finished.

Sorry, I missed that.
Should be all done now.

**Pull request merged**